sdk/cognitiveservices/azopenai/version.go

```diff
@@ -0,0 +1,11 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azopenai
+
+const (
+	version = "v0.1.0"
```

Comment:
Also need a CHANGELOG.md file with 0.1.0 inside. Also need a LICENSE.txt and a README file